service/core/client/src/main/scala/com/lightbend/lagom/internal/client/CircuitBreakersPanelInternal.scala

```diff
@@ -86,9 +86,7 @@ private[lagom] class CircuitBreakersPanelInternal(
 }
 
 @Singleton
-class CircuitBreakerConfig @Inject() (val configuration: Configuration) {
-
-  val config: Config = configuration.underlying.getConfig("lagom.circuit-breaker")
+class CircuitBreakerConfig @Inject() (val configuration: Config) {
```

Comment:
Despite this class being in an internal package, it looks like it is used by ConductR, so we will need to keep a deprecated constructor.